examples/intraday_examples/lead_time_analysis.py

```diff
@@ -0,0 +1,312 @@
+"""_summary_
+Lead time analysis for residual load, wind production, spv production and consumption:
+
+The user decides what forecast to use as benchmark forecast and what intraday forecast to use.
+The user also defines what issue date to look at from the benchmark forecast, the timestamp of interest (data time) and the time delta (how far back to look).
+This code then subtracts the estimated residual load, wind production, spv production and consumption from the benchmark forecast with the intraday forecast. This way we can investigate
+how the intraday forecast differs from the benchmark forecast for each release (each 15 min).
+
+
+Used libraries:
+volue_insight_timeseries: A library from volue used to access wapi. To install, run: pip install volue-insight-timeseries.
+pandas:                   A python library used to handle data. To install, run pip install pandas
+os:                       Functionalities for interacting with the operating system. This is a standard python library, so no installation is needed.
+plotly.express:           A module from plotly containing functions used to create graph plots. To install, run pip install plotly
+plotly.graph_objects:     Another module of the plotly library used to create plots.
+
+
+# User defined inputs:
+# area:                   The area of interest, eg. DE, PL etc.
+# benchmark_forecast:       'ec00', 'ec12', 'entsoe_da' or 'entsoe_intraday'
+# intraday_forecast:      'intraday' or 'intraday_lastec'
+# issue_date:             The date where the set1 curve (instance curve) was issued.
+# time_delta:             Defines how many hours of the intraday curve that is to be retrieved.
+# data_time:              The timestamp of the forecasted value.
+
+    """
+from volue_insight_timeseries import Session
+import pandas as pd
+import os
+import plotly.express as px
+import plotly.graph_objects as go
+
+# ***********************************
+# User defined inputs:
+area = "fr"
+benchmark_forecast = "ec00"
+intraday_forecast = "intraday"
+issue_date = '2023-11-26T00:00'
```

Comment:
Could you remind me why we're using pd.Timestamp for one timestamp and a string for another?